### PR TITLE
Improve Linux Install 4: Update PATH install section

### PR DESCRIPTION
**Pull request by wednesdei**
_Description of what this PR is changing or adding, and why:_

This PR updates "Update your path", of the Linux Installation Guide for Flutter. The update aims to make the instructions more explicit, ensuring they work across different shells and sessions. It also explains the necessity of updating the path and the impact on Flutter commands.

_Issues fixed by this PR (if any):_

This PR addresses issues related to shell discrepancies and the persistence of Flutter commands across sessions.

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/get-started/install/_path-linux-chromeos.md [hunk 1]
```diff
@@ -1,90 +1,76 @@
...
+```terminal
+$ echo $PATH
+```
+
+Verify that the `flutter` command is available by running:
+
+```terminal
+$ which flutter
+```
+{% include docs/dart-tool.md %}
```
Comment (atsansone) [inline src/get-started/install/_path-linux-chromeos.md:41]: **issues:**

1. Start any ordered list item with `1.`. This enables automatic numbering.
2. Indent code blocks under the list item.
3. Provide a response to all terminal commands.
4. Format CLI commands per [docopt](http://docopt.org/).

````suggestion
  ```terminal
  $ echo 'export PATH="$PATH:<path_to_flutter_directory>/flutter/bin"' >> $HOME/.bashrc
  ```

1. Reload the current shell profile.

  ```terminal
  source $HOME/.<rc file>
  ```

1. Verify that the `flutter/bin` directory exists in your PATH.

  ```terminal
  $ echo $PATH
  ```

  Response should resemble:
  
  ```terminal
  /usr/<example>/dev/flutter/bin:/usr/local/git/git-google/bin:/usr/local/git/current/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:
  ```

1. Verify that you can now use the `flutter` command.

  ```terminal
  $ which flutter
  ```

  Response should resemble:
  
  ```
  /usr/<example>/dev/flutter/bin/flutter
  ```

{% include docs/dart-tool.md %}
````

File: src/get-started/install/_path-linux-chromeos.md [hunk 2]
```diff
@@ -1,90 +1,76 @@
...
+
+In this command, replace `[PATH_TO_FLUTTER_GIT_DIRECTORY]` with your actual Flutter SDK path.
+
+#### Permanently update your path
+
+To permanently add Flutter to your path for _any_ terminal session, follow these steps:
+
+1. Determine the path of your Flutter SDK installation. You'll need this in Step 3.
+2. Open (or create) the `rc` file for your shell. This file runs whenever you open a new terminal window. If you're using the Bash shell, for example, edit `$HOME/.bashrc`.
+3. Add this line, replacing `[PATH_TO_FLUTTER_GIT_DIRECTORY]` with your Flutter SDK path:
```
Comment (atsansone) [inline src/get-started/install/_path-linux-chromeos.md:23]: **issues:**

1. Use [semantic breaks](https://sembr.org/).
2. Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

````suggestion
1. Find your Flutter SDK installation path.

  ```terminal
  $ find / -type d -name "flutter/bin" 2>/dev/null
  ```

  ```terminal
  /usr/<example>dev/flutter/bin
  ```

1. Append the following line to your `rc` shell file.
  Linux reads the `rc` shell "resource" file each
  time it opens a terminal.

  Replace `<path_to_flutter_directory>` with your Flutter path.
````

File: src/get-started/install/_path-linux-chromeos.md [hunk 3]
```diff
@@ -1,90 +1,76 @@
...
+```
+
+Verify that the `flutter` command is available by running:
+
+```terminal
+$ which flutter
+```
+{% include docs/dart-tool.md %}
+
+### Update the path directly
```
Comment (atsansone) [inline src/get-started/install/_path-linux-chromeos.md:43]: From here down, you can probably delete.

**Review by atsansone — CHANGES_REQUESTED** [review]
Good work! Just recommended some style issues and more concise commands.

File: src/get-started/install/_path-linux-chromeos.md (hunk 3 above)
Comment (wednesdei) [reply src/get-started/install/_path-linux-chromeos.md:43]: Thanks, I'll get started on fixing up these changes and resubmit, :)

**Review by wednesdei — COMMENTED** [review]
GTG

**Comment by atsansone** [comment]
This needs to wait until https://github.com/flutter/website/pull/9238 merges. It covers related material that reworks the underlying files involved in this PR.

**Review by atsansone — APPROVED** [review]

**Pull request merged**